plugin/kubectl-check_acr

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
is it a new tool, or required by above image

Reply:
This tool comes from https://github.com/yangl900/canipull/blob/main/plugin/kubectl-check_acr